### DISCUSSION TITLE
Welcome to LiteLED Discussions!

### DISCUSSION THREAD

**Discussion by Xylopyrographer** [Xylopyrographer/LiteLED#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AbdalrahmanMohammad** [comment]
thanks for your effort.
i have a problem using it..
when i make more than one instance, then if i control one of them they both get affected.
do you have any idea why that happens, and if i can solve this?

**Reply by Xylopyrographer** [reply]
I've not tested more than one instance (but not a bad idea).

Suggestion: try using the `new` keyword when creating the objects? Best I can think of off the top of my head.

Let me know how it goes.

**_EDIT:_**
Ah! Another thought. When creating the object, full syntax is:
`LiteLED( led_type, rgbw, channel );`

The `channel` is optional but defaults to `RMT_CHANNEL_0`. So if creating two instances, they would by default both use the same RMT channel. Try explicitly including a channel number and using a different channel number for the second instance.

See "Constructor" in the README for the channel number syntax.

**Reply by AbdalrahmanMohammad** [reply]
thanks for the prompt reply.
yeah that worked, it was all because i was using the same default channel.
my fault, keep up the good work.

**Reply by Xylopyrographer** [reply]
Super. Glad you got running!
